File: dayTwo/python/afterBreak/question_two.py
def score_grading(score_one, score_two, score_three):
    average = (score_one + score_two + score_three)/3;
    
    if(average >= 90 and average <= 100):
        print("A")
  
    elif(average >= 80 and average <= 90):
        print("B")
  
    elif(average >= 70 and average <= 80):
        print("C")
   
    elif(average >= 60 and average <= 70):
        print("D")
  
    else:
        print("F")
   
   
 
score_one = 60
score_two = 70
score_three = 50
    
score_grading(score_one, score_two, score_three)
    
 
    
       
     
    
    
    

File: dayTwo/javaScript/afterBreak/questionTwo.js

function scoreGrading(scoreOne, scoreTwo, scoreThree){
    let average = (scoreOne + scoreTwo + scoreThree)/3;
    
    if(average >= 90 && average <= 100){
        console.log("A");
    }
    else if(average >= 80 && average <= 90){
        console.log("B");
    }
     else if(average >= 70 && average <= 80){
        console.log("C");
    }
     else if(average >= 60 && average <= 70){
        console.log("D");
    }
     else if(average < 60){
        console.log("F");
     }
     
     
    }
   
   
  
scoreGrading()
let scoreOne = 60;
let scoreTwo = 70;
let scoreThree = 50;
    
scoreGrading(scoreOne, scoreTwo, scoreThree);
    
 
    
       
     
    
    
    
